Client.xcodeproj/project.pbxproj

```diff
@@ -3058,6 +3062,7 @@
 		D0FCF7EB1FE44DB6004A7995 /* AtDocumentEnd */ = {
 			isa = PBXGroup;
 			children = (
+				D000661320472890009BA6F6 /* __firefox__.js */,
```

Comment:
Do you know is this just adding these files to the project, or adding them to a target?